package.json

```diff
@@ -2,7 +2,7 @@
   "name": "ah-aquaman-frontend",
   "version": "1.0.0",
   "description": "",
-  "main": "index.js",
+  "main": "src/App.js",
```

Comment:
Why did you change the main from `index.js`

Reply:
Actually, this is not correct, thank you @AmosWels for pointing it out should be "src/index.js"